docs/structured-streaming-programming-guide.md

```diff
@@ -1689,6 +1689,23 @@ hence the number is not same as the number of original input rows. You'd like to
 There's a known workaround: split your streaming query into multiple queries per stateful operator, and ensure
 end-to-end exactly once per query. Ensuring end-to-end exactly once for the last query is optional.
 
+### State Store and task locality
+
+The stateful operations stores states for events in state stores of executors. State stores occupy resources such as memory and disk space to store the states.
```

Comment:
`The stateful operations stores` -> `The stateful operations store`?

Reply:
Fixed. Thanks.